core-feature-pack/common/src/main/resources/content/bin/common.bat

```diff
@@ -37,6 +37,7 @@ goto :eof
     echo "%~1" | findstr /I "\-\-add\-modules" > nul
     if errorlevel == 1 (
       rem Set default modular jdk options
+      set "DEFAULT_MODULAR_JVM_OPTIONS=!DEFAULT_MODULAR_JVM_OPTIONS! --enable-preview"
```

Comment:
it would be nice to keep the previous format

```
comment explaining need for the option
option
```

Reply:
Fixed @jbliznak